### DISCUSSION TITLE
Help adding username to filename?

### DISCUSSION THREAD

**Discussion by j-ktz** [UltimaHoarder/UltimaScraper#1074] [Q&A]
I've updated my config but it's not working, what am I doing wrong?
<img width="659" alt="screenshot 2021-04-29 at 2 54 44 PM" src="https://user-images.githubusercontent.com/59346628/116603436-ed369600-a8fa-11eb-8051-2be048b1b11d.png">

**Comment by UltimaHoarder** [comment]
Are you on Linux?

**Reply by j-ktz** [reply]
Nope, on a Mac.
